2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -56,7 +56,7 @@
  {
   "input": "But I never explained why this theorem is actually true, we only talked about what it's claiming.",
   "model": "nmt",
-  "translatedText": "Pero nunca expliqué por qué este teorema es realmente cierto, solo hablamos de lo que afirma.",
+  "translatedText": "Pero nunca he explicado porqué es cierto este teorema, solo hemos hablado de lo que afirma.",
```

Does this sound more natural in other places? We come from "hace un par de videos hablamos sobre ....", to me the old version sounds far more natural (it is more consistent with the simple verbal form used in the previous paragraph).

```suggestion
  "translatedText": "Pero nunca expliqué por qué es cierto este teorema, solo hablamos de lo que afirma.",
```

Regardless, you meant "por qué" and not "porqué":

```suggestion
  "translatedText": "Pero nunca he explicado por qué es cierto este teorema, solo hemos hablado de lo que afirma.",
```

Unless you meant to use "el porqué", but while correct, that start sounding a bit odd to my ears:

```suggestion
  "translatedText": "Pero nunca he explicado el porqué es cierto este teorema, solo hemos hablado de lo que afirma.",
```